ui/app/components/oidc/key-form.js

```diff
@@ -22,6 +22,7 @@ import { task } from 'ember-concurrency';
  * @param {Object} model - oidc client model
  * @param {onCancel} onCancel - callback triggered when cancel button is clicked
  * @param {onSave} onSave - callback triggered on save success
+ * @param {boolean} [limitOptions=null] - boolean to determine if form should render a limited set of options. Used when rendered outside of OIDC provider workflow (ex: SearchSelectWithModal when mounting a WIF secret engine).
```

Comment:
nit: the default here should probably match the type, if an explicit default isn't set then in the component I'd probably just leave this as `[limitOptions]` (since technically this would be `undefined` if the arg wasn't passed, but I think jsdoc is more useful for explaining when defaults are non-intuitive)
```suggestion
 * @param {boolean} [limitOptions=false] - boolean to determine if form should render a limited set of options. Used when rendered outside of OIDC provider workflow (ex: SearchSelectWithModal when mounting a WIF secret engine).
```